far/dirinfo.cpp

```diff
@@ -111,40 +111,44 @@ int GetDirInfo(string_view const DirName, DirInfoData& Data, multifilter* Filter
 	// Временные хранилища имён каталогов
 	string strLastDirName;
 	os::fs::find_data FindData;
+
+	// Mantis#0002692
+	struct SuspendMacros
+	{
+		SuspendMacros() { Global->CtrlObject->Macro.SuspendMacros(true); }
+		~SuspendMacros() { Global->CtrlObject->Macro.SuspendMacros(false); }
+	} Suspend;
```

Comment:
Minor: let's do it like in the other place (purely for consistency):
https://github.com/FarGroup/FarManager/blob/0b3d91dab6263574048368cb13f2f934f3521e0f/far/filelist.cpp#L7063-L7064